templates/contact/index.html

```diff
@@ -1,136 +1,246 @@
+                             name="utm_source"
+                             id="utm_source"
+                             value="" />
+                      <input type="hidden"
+                             aria-hidden="true"
+                             aria-label="hidden field"
+                             name="utm_content"
+                             id="utm_content"
+                             value="" />
+                      <input type="hidden"
+                             aria-hidden="true"
+                             aria-label="hidden field"
+                             name="utm_term"
+                             id="utm_term"
+                             value="" />
+                      <input type="hidden"
+                             name="returnURL"
+                             aria-label="returnURL"
+                             value="https://cn.ubuntu.com/contact/thank-you.html" />
+                    </li>
+                  </ul>
+                </fieldset>
+              </form>
 
-  <!-- /MARKETO FORM -->
+              <!-- /MARKETO FORM -->
+            </div>
+          </div>
+        </div>
       </div>
-     </div>
-    </div>
-  </div>
-</section>
+    </section>
 
-<section class="p-strip--light">
-  <div class="row">
-    <div class="col-12">
-      <h3 class="u-align-text--center">您可以通过以下方式关注我们</h3>
-    </div>
-  </div>
+    <section class="p-strip--light">
+      <div class="row">
+        <div class="col-12">
+          <h3 class="u-align-text--center">您可以通过以下方式关注我们</h3>
+        </div>
+      </div>
 
-  <div class="row p-divider">
-    <div class="col-4 p-divider__block">
-      <h4 align="center">官方微博</h4>
-    <div>
-        <a href="http://weibo.com/officialubuntu"> <img class="img1"  src="/static/img/social/weibo.png" alt="微博"></a>
-    </div>
-      <p align="center">
-        <a class="p-button--positive" href="http://weibo.com/officialubuntu" >点击关注</a></p>
-    </div>
+      <div class="row p-divider">
+        <div class="col-4 p-divider__block">
+          <h4 class="u-align--center">官方微博</h4>
+          <div>
+            <a href="https://weibo.com/officialubuntu">
+              <img src="https://assets.ubuntu.com/v1/b8c058fd-weibo.png" class="img1" alt="微博" />
+            </a>
+          </div>
+          <p class="u-align--center">
+            <a class="p-button--positive" href="https://weibo.com/officialubuntu">点击关注</a>
+          </p>
+        </div>
 
-    <div class="col-4 p-divider__block">
-      <h4 align="center">官方微信/企业微信</h4>
-      <img class="img1" id="wechat" src="/static/img/social/wechat.png"　alt="微信">
-      <p align="center">
-        <a href="#" class="p-button--base is--disabled" aria-disabled="true">手机微信扫一扫</a></p>
-    </div>
+        <div class="col-4 p-divider__block">
+          <h4 class="u-align--center">官方微信/企业微信</h4>
+          <img class="img1" id="wechat" src="https://assets.ubuntu.com/v1/6727c077-wechat.png"　alt="微信" />
```

Comment:
Slight refactoring in these three sections to appease djlint, added the assets to the asset manager instead of having them as static images and removed[ deprecated align property ](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/align). Everything else is linting